torch/_dynamo/guards.py

```diff
@@ -591,6 +595,15 @@ def BACKEND_MATCH(self, guard: Guard):
         ]
         self._produce_guard_code(guard, code)
 
+    def CONFIG_HASH_MATCH(self, guard: Guard):
+        """Guard on the hash of the compiled function's dynamo config"""
+
+        assert guard.source is GuardSource.GLOBAL
+        code = [
+            f"___applicable_config_hash() == '{torch._dynamo.eval_frame.get_saved_else_current_config_hash().hex()}'"
```

Comment:
why are you hex'ing it 🤔

Reply:
Well the choice is between storing the hex value and storing the bytes. I went with storing the bytes but making the guard more human-readable with the hex :sweat_smile:

Reply:
If you repr the bytes string it will still be human readable, but you still avoid unnecessary comparing a lot of extra bytes you don't need

Reply:
If this is what you mean by human readable, I unfortunately wouldn't agree? 

```
b'\x92\xec>\xbc\x03\x03\x8c Ol\xd3\xa7\xa2M]0'
```

Reply:
It's ascii, and it uniquely determines the hash, what more can you be looking for :P